library/ziti.c

```diff
@@ -206,14 +212,72 @@ int ziti_init_opts(ziti_options *options, uv_loop_t *loop) {
     return ZITI_OK;
 }
 
-static void ziti_init_async(uv_async_t *ar) {
-    struct ziti_init_req *init_req = ar->data;
+extern bool ziti_is_enabled(ziti_context ztx) {
+    return ztx->enabled;
```

Comment:
should this guard against a uninitialized/null'ed context? it'd be friendlier to not crash imo... but maybe that's just me? :)

Reply:
just you

otherwise we need to do it everywhere